docs/components/paragraphs.md

```diff
@@ -8,7 +8,8 @@ description: >-
   This page is under active development. Information is being moved into it from
   these pages:
 
-  * "http://cfpb.github.io/design-manual/brand-guidelines/typography.html
+
+  * http://cfpb.github.io/design-manual/brand-guidelines/typography.html
```

Comment:
These could be updated to be clickable links, but otherwise this is a fine edit.